pythia6.sh

```diff
@@ -1,59 +1,77 @@
+# a pythia6 recipe based on the one from FairROOT
```

Comment:
How about we create alisw/pythia6 and patch it there, rather than doing everything in the recipe?
That would better mark our changes.